specification/monitor/resource-manager/Microsoft.Insights/stable/2020-10-01/activityLogAlerts_API.json

```diff
@@ -0,0 +1,546 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure Activity Log Alerts API",
+        "description": "API for Azure Activity Log Alert rules CRUD operations.",
+        "version": "2020-10-01",
+        "x-ms-code-generation-settings": {
+          "name": "MonitorManagementClient"
+        }
+    },
+    "host": "management.azure.com",
+    "schemes": [
+      "https"
+    ],
+    "consumes": [
+      "application/json"
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "security": [
+      {
+        "azure_auth": [
+          "user_impersonation"
+        ]
+      }
+    ],
+    "securityDefinitions": {
+      "azure_auth": {
+        "type": "oauth2",
+        "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+        "flow": "implicit",
+        "description": "Azure Active Directory OAuth2 Flow",
+        "scopes": {
+          "user_impersonation": "impersonate your user account"
+        }
+      }
+    },
+    "paths": {
```

Comment:
@MyronFanQiu Can you please help me understand what I need to do to fix this issue?
![image](https://user-images.githubusercontent.com/48387592/96600027-2e6d3e00-12f9-11eb-9615-78097e9a1074.png)

![image](https://user-images.githubusercontent.com/48387592/96600062-375e0f80-12f9-11eb-870f-9b640441a057.png)